CameraOverlay/OrchestratorClass.py

```diff
@@ -0,0 +1,79 @@
+"Orchestrator Script That Controls The Camera System"
+
+#imports
+import json
+import argparse
+import time
+import paho.mqtt.client as mqtt
+import config
+
+#Get arguments passed into Python script
+
+def get_args(argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host", type=str,
+                        default="192.168.100.100",
+                        help="ip address of the broker")
+    return parser.parse_args(argv)
+
+#Orchestrator Class
```

Comment:
Probably an unnecessary comment, as we know that this is the `Orchestrator` class already